roles/nginxplus/files/conf/http/cdh_geniza_prod.conf

```diff
@@ -46,6 +46,16 @@ server {
         proxy_set_header X-Real-IP $remote_addr;
         proxy_cache geniza_prodcache;
         health_check interval=10 fails=3 passes=2;
+
+        # allow princeton network
+        allow 128.112.0.0/16;
```

Comment:
can you replace these 4 lines with `include /etc/nginx/conf.d/templates/restrict.conf;`

Reply:
yes, sure!